<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanSetupCommonWidget</class>
 <widget class="QWidget" name="ScanSetupCommonWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Scan control setup</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Lock timeout [ms]:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="lockTimeout">
        <property name="maximum">
         <number>2000000</number>
        </property>
        <property name="value">
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>PAT timeout [ms]:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="patTimeout">
        <property name="maximum">
         <number>2000000</number>
        </property>
        <property name="value">
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Manual stepping:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="manualStepping">
        <property name="toolTip">
         <string>Frequency stepping during scan is done manually</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="toolTip">
         <string>If selected, when starting a new step, the scan procedure will ask for PIDs 0, 16, 17, 18 and 20. Otherwise only PID 0 is asked.</string>
        </property>
        <property name="text">
         <string>Use default DVB PIDs:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="defaultDvbPIDs">
        <property name="toolTip">
         <string>If selected, when starting a new step, the scan procedure will ask for PIDs 0, 16, 17, 18 and 20. Otherwise only PID 0 is asked.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
